Extension/package.json

```diff
@@ -1472,11 +1474,13 @@
     "minimatch": "~3.0.4",
     "mkdirp": "~0.5.1",
     "tmp": "~0.0.33",
-    "vscode-cpptools": "2.1.1",
+    "vscode-cpptools": "^2.1.2",
     "vscode-debugadapter": "~1.33.0",
     "vscode-debugprotocol": "~1.33.0",
     "vscode-extension-telemetry": "^0.1.1",
     "vscode-languageclient": "5.1.1",
+    "webpack": "^4.29.6",
+    "webpack-cli": "^3.2.3",
```

Comment:
Do these additions generate a new 3rd party licenses file that we need to check in?

Reply:
These should actually be under devDependencies. I don't think we have to ship any webpack code.

Reply:
Oops, I didn't see that this change wasn't made. I'll submit a fix.